documentation/index.rst

```diff
@@ -10,6 +10,7 @@ available in the command line interface.
    :maxdepth: 1
    :caption: Content:
 
+   content/developmentworkflow
```

Comment:
I think this line makes "Development workflow" become the first item in kw's documentation list. It think it makes sense for the development workflow to come in the last place.